client/analytics/components/report-chart/index.js

```diff
@@ -57,7 +57,7 @@ export class ReportChart extends Component {
 	getChartMode() {
 		const { filters, query } = this.props;
 		if ( ! filters ) {
-			return null;
+			return;
```

Comment:
It looks like you'll need to fix a unit test with `undefined` v. `null`.

Reply:
Completely missed that one. Thank you!